analyzers/tests/SonarAnalyzer.UnitTest/TestSuiteInitialization.cs

```diff
@@ -38,6 +40,14 @@ public static void AssemblyInit(TestContext context)
 
             var vbVersions = ParseOptionsHelper.Default(LanguageNames.VisualBasic).Cast<VisualBasicParseOptions>().Select(x => x.LanguageVersion.ToDisplayString());
             Console.WriteLine(@"VB.Net versions used for analysis: " + string.Join(", ", vbVersions));
+
+            // The AltCover tool has a limitation. It has to be invoked without a parameter for the project/solution path.
+            // Due to this we have to call it from the analyzers folder and the working directory is different when running in CI context.
+            if (!File.Exists(Path.Combine(Paths.RelativePathToTestProjectRoot, "nuget.config")))
+            {
+                Paths.UpdateRelativePathToTestProjectRoot(@"..\..\..\..\");
+            }
```

Comment:
It's weird that half of the logic is in `Paths.cs` (the default value) and the other half is here. They should be together (here or there, probably there to have all "path" logic together. In some "InitPaths()" method. To reuse the local scoped knowledge once we build more paths).